graylog2-rest-models/src/main/java/org/graylog2/rest/models/system/indexer/responses/IndexInfo.java

```diff
@@ -37,8 +38,9 @@
     @JsonProperty
     public abstract boolean isReopened();
 
-    public static IndexInfo create(IndexStats primaryShards,
-                                   IndexStats allShards,
+    @JsonCreator
+    public static IndexInfo create(@JsonProperty("primary_shards") IndexStats primaryShards,
+                                   @JsonProperty("all_shards") IndexStats allShards,
                                    List<ShardRouting> routing,
```

Comment:
If this should ever be deserialized from an actual JSON payload, we're probably missing the `@JsonProperty` annotations for the last 2 method parameters. Otherwise the `@JsonProperty` annotations for the first 2 method parameters are obsolete.